lib/java/src/main/java/org/apache/thrift/meta_data/FieldMetaData.java

```diff
@@ -103,8 +105,12 @@ public static Map<? extends TFieldIdEnum, FieldMetaData> getStructMetaDataMap(
                 + sClass.getName()
                 + ", message: "
                 + e.getMessage());
+      } catch (ReflectiveOperationException e) {
```

Comment:
I think this could be cleaned up further. This ReflectiveOperationException also replaces the above two individual catch clauses, which are both ReflectiveOperationExceptions.

Reply:
@ctubbsii  thanks for the catch - fixed now. must've been due to merge conflict.